treescript/src/treescript/github/versionmanip.py

```diff
@@ -129,7 +147,6 @@ async def do_bump_version(client: GithubClient, files: List[str], next_version:
             if contents == new_contents:
                 raise Exception("File was not changed!")
             changes[file_] = new_contents
+            diff += unified_diff(file_contents[file_].splitlines(), new_contents.splitlines(), fromfile=file_, tofile=file_, lineterm="")
```

Comment:
Question: I remember Treescript outputs the diff as an artifact. Does this implementation ensures we still store the diff somewhere?

Reply:
Oh hm, no this isn't adding the diff as an artifact, but it is showing it in the log, e.g:
https://firefox-ci-tc.services.mozilla.com/tasks/ARx6P9gKSJOtiT20GrPKjQ/runs/1/logs/public/logs/live_backing.log

Is that good enough do you think? One problem is that because we no longer have a repo, I don't think that diff will actually apply if you attempt to import it, so I don't know if having it as an artifact still makes sense or not.